src/picasso-components/react-components/navigation-panel.jsx

```diff
@@ -20,32 +20,36 @@ const NavigationButton = ({ ...rest }) => {
           width: `${width}px`,
           height: `${width}px`,
           '&:focus': {
-            background: 'rgba(240, 240, 240, 0.5)',
+            color: '#111111',
```

Comment:
Better to get a grey scale from sprout, etc `#1A1A1A`. https://github.com/qlik-trial/sprout/blob/main/src/theme/internal/variables.ts